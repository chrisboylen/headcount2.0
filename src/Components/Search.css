form {
  display: flex;
  justify-content: center;
  margin: 0 0 2em 0;
}

input {
  background-color: #355053;
  border: solid 4px #f7b488;
  ;
  border-radius: 7px;
  color: #f7b488;
  font-size: 2em;
  height: 2em;
  width: 20em;
  margin: auto;
  padding: 0 1em 0 1em;
}

input:focus {
  outline: none;
}