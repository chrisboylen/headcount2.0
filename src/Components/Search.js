import React from 'react';

class Search extends Component {
  constructor() {
    super();
    this.state = {
      input: ''
    }
  }

  handleSubmit = (e) => {
    this.setState({ input: e.target.value })
  }

  render() {
    return (
      <form onSubmit={(e) => this.handleSubmit(e)}>
        <input type="text"
          value={ this.state.input }
          onChange={ (e) => this.handleChange(e) }
        />
      </form>
    )
  }
}