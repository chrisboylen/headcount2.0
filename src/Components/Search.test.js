import React from 'react';
import { shallow } from 'enzyme';
import Search from './Search';

describe('SEARCH', () => {
  it('Should exist', () => {
    const wrapper = shallow(<Search />)

    expect(wrapper).toBeDefined();
  })

  it('Should match the snapshot', () => {
    const wrapper = shallow(<Search />);

    expect(wrapper.html()).toMatchSnapshot()
  })
})