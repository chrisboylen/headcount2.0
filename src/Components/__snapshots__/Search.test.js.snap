// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SEARCH Should match the snapshot 1`] = `"<form><input type=\\"text\\" placeholder=\\"Search\\"/></form>"`;
